wifiphisher/pywifiphisher.py

```diff
@@ -53,8 +53,14 @@ def parse_args():
     parser.add_argument(
         "-iI",
         "--internetinterface",
-        help=("Choose an interface that is connected on the Internet" +
+        help=("Choose an interface that is connected on the Internet. " +
               "Example: -iI ppp0"))
+    parser.add_argument(
+        "-pI",
+        "--protectinterface",
+        nargs='+',
+        help=("Choose an interface that you want to protect and persist its connection. " +
```

Comment:
The wording needs to be changed a bit to show you can have more than one interface.